pkg/ocm/versions.go

```diff
@@ -313,12 +313,21 @@ func IsValidVersion(userRequestedVersion string, supportedVersion string, cluste
 		return false, err
 	}
 
+	c, err := ver.NewVersion(clusterVersion)
+	if err != nil {
+		return false, err
+	}
+
 	isPreRelease := a.Prerelease() != "" && b.Prerelease() != ""
 
+	// User wants to upgrade from a prerelease version
+	// i.e. from 4.14.0-rc.4 to 4.10.0
+	fromPreRelease := c.Prerelease() != ""
```

Comment:
can we use the change as below?
isPreRelease := a.Prerelease() != "" && b.Prerelease() != "" || c.b.Prerelease() != ""